typescript/cli/src/context/context.ts

```diff
@@ -93,10 +93,8 @@ export async function signerMiddleware(argv: Record<string, any>) {
       const metadata = multiProvider.getChainMetadata(chain);
 
       if (hasProtocol(protocol))
-        altVmProvider.set(
-          chain,
-          await getProtocolProvider(protocol).createProvider(metadata),
-        );
+        altVmProvider[protocol] =
```

Comment:
We still need providers to be created for every chain because they use chain metadata to get the RPC URLs, etc.